content/guides/repl/data_visualization_at_the_repl.adoc

```diff
@@ -377,8 +377,8 @@ Here's an example program which demonstrates these sort of Exceptions.
 ----
 
 We don't know it yet, but our `average` function fails when applied to an empty collection of numbers.
-However, visualizing the Exception makes it easy to diagnose. In the below REPL session, we can see 
-that we calling our function with an empty vector of numbers led to dividing zero by zero:
```

Comment:
"us calling" would be correct, but I don't think either "us" or "we" adds anything.